interface/src/Application.cpp

```diff
@@ -7530,6 +7532,10 @@ void Application::registerScriptEngineWithApplicationServices(const ScriptEngine
     scriptEngine->registerGlobalObject("HifiAbout", AboutUtil::getInstance());  // Deprecated.
     scriptEngine->registerGlobalObject("ResourceRequestObserver", DependencyManager::get<ResourceRequestObserver>().data());
 
+    
+    scriptEngine->registerGlobalObject("ExternalResource", ExternalResource::getInstance());
```

Comment:
if we want QML to have access to this, I think it also needs to be added in `onDesktopRootContextCreated` and `setupQmlSurface`

Reply:
Ok, tried that. How should it be tested to make sure it went right?